src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,181 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User userIsNull;
+    private User userLoginIsNull;
+    private User userLoginIs5Char;
+    private User userLoginIs6Char;
+    private User userLoginIs7Char;
+    private User userPassIsNull;
+    private User userPassIs5Char;
+    private User userPassIs6Char;
+    private User userPassIs7Char;
+    private User userAgeIsNull;
+    private User userAgeIs17;
+    private User userAgeIs18;
+    private User userAgeIs19;
+    private User userOk1;
+    private User userOk2;
+    private User userOk3;
+    private User userOk4;
+    private User userOk5;
+    private User userOk6;
+    private User theSameUserOk1;
+    private User theSameUserOk3;
```

Comment:
There is no need for all these users. It is enough to make one and initialize it with the necessary values in each test

Reply:
Ok)